docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,41 @@
+{#
```

Comment:
We're in the process of changing Terra's autodoc templates to not be like this. Right now, these will create a dedicated HTML page per method, which results in _huge_ websites / lots of HTML pages. That makes the docs build really slow. And we think it's also not a great user experience to have to change the page so frequently. Compare to e.g. Python docs having all the functions and methods of a module on the same page.

Instead, I recommend using templates like what I have here: https://github.com/Qiskit/qiskit_sphinx_theme/pull/458/files#diff-e3f9c219c6e6c2e12d14ace3f19390fcf4666eb456ab90012b7f1165c9a33e32

There's a chance these will be further improved the next few days as I switch Terra, specifically to have better filtering of the methods and attributes. But this should be fine for now and I can DM you when we have a better template.

Reply:
Sounds good